go.mod

```diff
@@ -1,8 +1,12 @@
 module github.com/DIMO-Network/rewards-api
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
I think we can delete the toolchain piece sometime this get randomly added when I run tidy as well